app/assets/javascript/components/editor/editor.js

```diff
@@ -0,0 +1,145 @@
+import { Controller } from '@hotwired/stimulus';
+import DOMPurify from 'dompurify';
+
+const EditorController = class extends Controller {
+  static targets = ['editor', 'toolbar', 'formInput'];
+
+  static TOOLBAR_ACTIONS = {
+    ul: 'insertUnorderedList',
+  };
+
+  static ALLOWED_TAGS = ['p', 'ul', 'li', '#text'];
+
+  connect() {
+    this.enabled = true;
+
+    // this will disable IE11 but make sure enabled for JS tests
+    if ((typeof process !== 'undefined' && process.env.NODE_ENV !== 'test') || typeof window.ClipboardEvent === 'undefined') {
```

Comment:
Move the first part of this statement to the tests. This script should not be concerned with the unit tests.